src/main/java/net/md_5/specialsource/Jar.java

```diff
@@ -75,12 +75,26 @@
      * @return
      */
     public boolean containsClass(String clazz) {
-        try {
-            return contains.contains(clazz) ? true : getClass(clazz) != null;
-        } catch (IOException ex) {
-            // IO error - regardless we do not have access to the class, so we can ignore it and move on
```

Comment:
This code no longer has the same behaviour as before - if we would ever actually get to this point, we will now incorrectly return true.

Shouldn't happen unless the jar file is modified or deleted while we're using it.